cypress/pages/new-transaction.page.ts

```diff
@@ -0,0 +1,84 @@
+import { homeLocator } from "../pages/home.page";
+
+const transactionPageLocator = {
+  amount: "#amount",
+  note: "#transaction-create-description-input",
+  payButton: "[data-test='transaction-create-submit-payment']",
+  userBalance: "[data-test='sidenav-user-balance']",
+  paymentMessage: ".MuiTypography-gutterBottom",
+  requestButton: "[data-test='transaction-create-submit-request']",
+  amountHelperText: "#transaction-create-amount-input-helper-text",
+  noteHelperText: "#transaction-create-description-input-helper-text",
+};
+
+export const transactionPage = {
+  open: () => {
+    cy.get(homeLocator.newTransactionButton).click();
+    cy.location("pathname").should("equal", "/transaction/new");
+  },
+  chooseBeneficiary: () => {
+    cy.get("ul li:first").click();
```

Comment:
Are you sure you would like to leave this css selector as it is? I think its still better to move it to some variable in this file